modules/repository-azure/build.gradle

```diff
@@ -17,32 +17,47 @@ esplugin {
 }
 
 versions << [
-  'azure': '12.20.1',
-  'azureCommon': '12.19.1',
-  'azureCore': '1.34.0',
-  'azureCoreHttpNetty': '1.12.7',
-  'azureJackson': '2.13.4',
-  'azureJacksonDatabind': '2.13.4.2',
-  'azureAvro': '12.5.3',
-
-  'jakartaActivation': '1.2.1',
-  'jakartaXMLBind': '2.3.2',
+  //see https://repo1.maven.org/maven2/com/azure/azure-storage-blob/ for transitive dependencies
+  'azure': '12.26.1', // <-- when upgrading versions start start here and manually upgrade the transitive dependencies
+  'azureCommon': '12.25.1',
+  'azureCore': '1.49.1',
+  'azureCoreHttpNetty': '1.15.1',
+  'azureJackson': '2.15.4', // keep this roughly in sync with the jackson version in the rest of the codebase
```

Comment:
All of the versions are manually pulled from the respective version POM's, except for this one. The motivation for this upgrade is bring jackson up to at least 2.15.0 to better match the rest of the jackson versions and avoid the dependency from showing up in security scans (even though we are not impacted by what it is scanning for).  I could in theory, only upgrade Jackson since either way we would use mismatched transitive versions, but figured while we are upgrading this module we should just upgrade all the versions here.